nominees/project-konko.json

```diff
@@ -11,25 +14,23 @@
   "SDGs": [
     {
       "SDGNumber": 4,
-      "evidenceURL": "https://www.unicef.org/innovation/FundGraduate/NubianVR \n https://medium.com/making-nubianvr/nubianvrs-educational-vision-the-vision-the-road-map-7fd59ec8d486"
+      "evidenceText": "Using Virtual Reality as a Tool to Improve Instruction Quality"
     }
   ],
-  "sectors": [],
+  "sectors": [
+    "Education"
+  ],
   "type": [
     "software"
   ],
   "repositoryURL": "https://github.com/nubianvr-source/ProjectKonko",
   "organizations": [
     {
-      "name": "UNICEF Ventures Fund",
```

Comment:
I would preserve this additional entry in the orgs array

Reply:
Resolved in commit aefad21

Reply:
Thank you!